Articles/Blog/2020-03-software-communities.md

```diff
@@ -0,0 +1,185 @@
+# Scientific Software Projects and Their Communities
+
+#### Contributed by [Rene Gassmoeller](https://github.com/gassmoeller "Rene Gassmoeller GitHub Profile")
+
+#### Publication date: March 20, 2020
+
+In the past several years, we have seen a growing consensus that improving the state of
+scientific software requires focusing on the people behind the software. This
+is particularly true for the [software engineers and
+scientists](https://bssw.io/blog_posts/us-research-software-engineer-us-rse-association)
+developing the software and for the [maintainers and leaders](https://bssw.io/blog_posts/leading-a-scientific-software-project-it-s-all-personal)
+of projects, but also for whole projects improving their [software development
+processes](https://bssw.io/blog_posts/productivity-and-sustainability-improvement-planning-psip).
+These people are not just individuals, however; when they work together on a
+software project, they form a community. We have learned a lot about
+user/developer communities from open-source software projects (see
+[opensource.guide](https://opensource.guide/building-community/)), yet we know
+relatively little about the challenges that are specific to scientific
+software.
+
+### Community engagement as a limit to project growth?
+
+I am a scientist who develops software, and I regularly meet with others in my field (geophysics) to improve their software development
+processes. During my work, I have frequently noticed that some software project members have
+valuable scientific ideas and follow best practices for software design yet
+never attract a sufficient user base to establish themselves. 
+(%GAIL asks- not sure wheat you mean by "establish themselves" - I guess "as a thriving community" but that seems repetitious, and establish themselves as scientists is not what you mean, I am sure)
```

Comment:
I mean "establish themselves" as in "establish themselves as ongoing projects with thriving communities." which would require rephrasing the next sentence. I will leave the decision up to @bernhold if this needs clarification here, I am fine modifying it.

Reply:
I added "their software project" to clarify without redundancy